src/components/ChooseJobProfile/ChooseJobProfile.js

```diff
@@ -0,0 +1,120 @@
+import React, {
+  useState,
+  useEffect,
+  useRef,
+} from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import {
+  history as historyShape,
+  location as locationShape,
+} from 'react-router-prop-types';
+
+import { stripesConnect } from '@folio/stripes/core';
+import { ConfirmationModal } from '@folio/stripes/components';
+import SafeHTMLMessage from '@folio/react-intl-safe-html';
+import {
+  JobProfiles,
+  getJobProfilesColumnProperties,
+  DEFAULT_JOB_PROFILES_COLUMNS,
+  getJobProfilesItemFormatter,
+} from '@folio/stripes-data-transfer-components';
+
+import { manifest as jobProfilesManifest } from './manifest';
+
+const customProperties = getJobProfilesColumnProperties({
+  columnWidths: { description: '40%' },
+  columnMapping: { description: <FormattedMessage id="ui-data-export.description" /> },
+  visibleColumns: [
+    DEFAULT_JOB_PROFILES_COLUMNS.NAME,
+    'description',
+    DEFAULT_JOB_PROFILES_COLUMNS.UPDATED,
+    DEFAULT_JOB_PROFILES_COLUMNS.UPDATED_BY,
+  ],
+});
+
+const formatter = getJobProfilesItemFormatter({});
+
+const ChooseJobProfileComponent = ({
+  resources,
+  mutator,
+  history,
+  location,
+}) => {
+  const [isConfirmationModalOpen, setConfirmationModalState] = useState(false);
+  const [selectedProfile, setSelectedProfile] = useState({});
+  const fileDefinitionIdRef = useRef(null);
+
+  useEffect(() => {
+    fileDefinitionIdRef.current = location.state?.fileDefinitionId;
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
+
+  return (
+    <>
+      <JobProfiles
```

Comment:
It seems, if you are going to use it here, JobProfiles shouldn't be placed in the settings folder any more.

Reply:
I am not using JobProfiles from settings but from the shared components.